03-leadership-and-inclusion/code-journals.md

```diff
@@ -0,0 +1,43 @@
+# Code Journals
+
+## Purpose
+
+The purpose of the Code Journal is to give each individual at least one tool for self-reflection. With the practice of consistently filling out the code journal, we hope to accomplish the following specific things:
+
+- Teach and practice self-reflection
+- Reinforce weekly learning goals
+- Give you an artifact to catalog and review your learnings over time
+
+**An Ada instructor will never ask to see your code journal.**
+
+## How
+
+The code journal is yours. Feel free to fill it out and use it however is most useful for you. Use markers and draw and bullet journal if that's what works best for you! Keep it clean and orderly if that's what you'd like, or scribble in it and draw pictures if you need to.
+
+At Ada, we'll ask that each person is intentional about how they reflect, though. It may be useful to use the following template for your weekly entry. This template emphasizes reflecting on the learning goals.
+
+---
+
+**Week #X**
+
+Question | My Response |
+--- |
+fill in Learning Goal #0 | fill in your reflection on Learning Goal #0
+... | ...
+fill in Learning Goal #(X-1) | fill in your reflection on Learning Goal #(X-1)
+What went well for me this week? |
+What concepts was the most difficult for me this week? |
+What are things I'd like to keep doing? |
+What are things I'd like to change or start doing? What are things I need to do better on? |
```

Comment:
I think that these are the conclusions that we want them to be able to draw, but I think it might be good to be able to build up to those conclusions. I've just briefly linked a few additional resources that do a bit more deep-diving, so maybe we could just list out prompts like this or link to these resources (if you think they're valuable)? 

http://peterpappas.com/2010/01/reflective-student-taxonomy-reflection-.html
http://mrrobinsonau.blogspot.ca/2009/04/reflective-writing-sentence-starters.html